ts_scripts/install_dependencies.py

```diff
@@ -205,6 +207,20 @@ def install_numactl(self):
 
     def install_cpp_dependencies(self):
         if os.system("clang-tidy --version") != 0 or args.force:
+            # Enable installation of latest cmake release
+            # Ref: https://apt.kitware.com/
+            os.system(
+                f"{self.sudo_cmd}apt-get install -y ca-certificates gpg lsb-release"
+            )
+            os.system(
+                f"wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | gpg --dearmor - | {self.sudo_cmd} tee /usr/share/keyrings/kitware-archive-keyring.gpg >/dev/null"
+            )
+            os.system(
+                f'echo "deb [signed-by=/usr/share/keyrings/kitware-archive-keyring.gpg] https://apt.kitware.com/ubuntu/ $(lsb_release -cs) main" | {self.sudo_cmd} tee /etc/apt/sources.list.d/kitware.list >/dev/null'
+            )
+            os.system(f"{self.sudo_cmd}apt-get update")
```

Comment:
why these are added at here?

Reply:
To enable installing a newer `cmake` version on linux. The latest version available on the default apt repositories is `3.16.3`. Build will fail with the following error:
```
CMake 3.26.4 or higher is required.  You are running version 3.16.3
```

Reply:
CA certificate should be dockerfile, not here.